mintpy/timeseries_rms.py

```diff
@@ -222,8 +222,8 @@ def main(iargs=None):
     (inps.rms_list,
      inps.date_list,
      inps.rms_file) = ut.get_residual_rms(inps.timeseries_file,
-                                          mask_file=inps.maskFile,
-                                          ramp_type=inps.deramp)
+                                         mask_file=inps.maskFile,
+                                         ramp_type=inps.deramp)
```

Comment:
The changes in this file seem unnecessary, could we undo them?

Reply:
Sure. I am sorry. This is my VSCode automatically identifies and removes the spaces that are not consistent with the other indentations.